src/lib/utils/os_utils.cpp

```diff
@@ -655,6 +665,16 @@ void OS::free_locked_pages(const std::vector<void*>& pages)
       }
    }
 
+void OS::page_named(void* page, size_t size)
+   {
+   constexpr char name[] = "Botan";
+#if defined(BOTAN_TARGET_OS_HAS_PRCTL)
+   (void)prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, reinterpret_cast<uintptr_t>(page), size, name);
+#else
+   (void)name;
```

Comment:
Use `BOTAN_UNUSED` here

Reply:
Should be `BOTAN_UNUSED(name, size)` since both otherwise MSVC will warn about unused params.